scripts/environment-deploy.sh

```diff
@@ -53,6 +53,56 @@ pushd "$SOLUTION_DIR/post-deployment" > /dev/null
 $EXEC sls invoke -f postDeployment -l -s "$STAGE"
 popd > /dev/null
 
+# Get the first header (not Changelog) in CHANGELOG.md
+versionLine="$(cat CHANGELOG.md | grep -m 1 "[0-9]\.[0-9]\.[0-9]\|Beta")"
+
+# Get version number
+versionNumber="$(echo $versionLine | grep -o "[0-9]\.[0-9]\.[0-9]\|Beta" | head -n 1)"
+
+# Get version date (or generate if beta)
+if [ "$versionNumber" == "Beta" ]
+then
+    # instead of showing a date in the beta condition, show the latest release version
+    latestReleaseVersion="$(cat CHANGELOG.md | grep -o "[0-9]\.[0-9]\.[0-9]" | head -n 1)"
+    versionDate="Latest Release Version: $latestReleaseVersion"
+else
+    versionDate="$(echo $versionLine | grep -o "[0-9][0-9][0-9][0-9]\-[0-9][0-9]\-[0-9][0-9]")"
+fi
+
+# Is there a stage.yml file?
+FILE=main/config/settings/${STAGE}.yml
+if [ -f "$FILE" ]
+then
+    # Yes-->Is there a versionDate and versionNumber key?
+    if (cat "$FILE" | grep -q "versionDate") && (cat "$FILE" | grep -q "versionNumber")
+    then
+        # Yes-->Are they different from above?
+        oldVersionNumber="$(cat "$FILE" | grep -o "[0-9]\.[0-9]\.[0-9]\|Beta" | head -n 1)"
```

Comment:
Same here, we might need to extend the first number to account for more than one digit.

Reply:
Yup, I will put `[0-9]+` for all the release number `grep` calls

Reply:
Had to escape the special character so it was `[0-9]\+` for each number in `<number>.<number>.<number>`